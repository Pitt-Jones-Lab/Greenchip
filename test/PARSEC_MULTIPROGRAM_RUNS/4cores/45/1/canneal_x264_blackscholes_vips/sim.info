{'benchmarks_revision': ('cat: .gitid: No such file or directory',),
 'benchmarks_root': ('/homes/grads/dek61/SNIPER/benchmarks',),
 'cmdline': ['/homes/grads/dek61/SNIPER/sniper/run-sniper',
             '-n',
             '4',
             '-m',
             'localhost',
             '-d',
             '/scratch/nbp3/PARSEC_MUTLIPROGRAM_SHELL/4core/45/1/canneal_x264_blackscholes_vips',
             '-c',
             '/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_45_1_4.cfg',
             '-s',
             'stop-by-icount:10000000000',
             '--curdir=/scratch/nbp3/PARSEC_MUTLIPROGRAM_SHELL/4core/45/1/canneal_x264_blackscholes_vips',
             '--trace-manual',
             '-g',
             '--traceinput/enabled=true',
             '-g',
             '--traceinput/emulate_syscalls=true',
             '-g',
             '--traceinput/num_apps=4',
             '-g',
             '--traceinput/trace_prefix=/tmp/tmpO3JlWG/run_benchmarks'],
 'git_revision': '68418dbe7cab15c65f1f4d300da0c505329beedd',
 'host': 'sb4',
 'pin_version': 'unknown',
 'rusage': (20009.354114999998,
            1213.442528,
            307880,
            0,
            0,
            0,
            74610,
            2,
            0,
            10752,
            1544,
            0,
            0,
            0,
            132539145,
            6666967),
 'snipercmd': ['bash',
               '-c',
               '/homes/grads/dek61/SNIPER/sniper/lib/sniper -c /homes/grads/dek61/SNIPER/sniper/config/base.cfg --general/total_cores=4 --general/output_dir=/scratch/nbp3/PARSEC_MUTLIPROGRAM_SHELL/4core/45/1/canneal_x264_blackscholes_vips --config=/homes/grads/dek61/SNIPER/sniper/config/nehalem45.cfg --config=/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_45_1_4.cfg --traceinput/enabled=true --traceinput/emulate_syscalls=true --traceinput/num_apps=4 --traceinput/trace_prefix=/tmp/tmpO3JlWG/run_benchmarks -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/scratch/nbp3/PARSEC_MUTLIPROGRAM_SHELL/4core/45/1/canneal_x264_blackscholes_vips/sim.scripts.py -g --hooks/script0args='],
 't_elapsed': 9666.978364944458,
 't_start': 1463066061.2264211,
 'tracecmds': [],
 'user': 'nbp3',
 'vmem': 1734959104L}
