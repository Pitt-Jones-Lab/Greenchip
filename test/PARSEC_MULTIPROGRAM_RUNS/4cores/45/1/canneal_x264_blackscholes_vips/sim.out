                                   | Core 0      | Core 1      | Core 2      | Core 3     
  Instructions                     |  1707310513 |  2523139546 |  2819695727 |  2948496162
  Cycles                           |  1705154430 |  1705154430 |  1705154430 |  1705154430
  IPC                              |        1.00 |        1.48 |        1.65 |        1.73
  Time (ns)                        |   641035500 |   641035500 |   641035500 |   641035500
  Idle time (ns)                   |       93258 |     1499234 |      268123 |      846225
  Idle time (%)                    |        0.0% |        0.2% |        0.0% |        0.1%
Branch predictor stats             |             |             |             |            
  num correct                      |   167846255 |   137509752 |   334673489 |   251806602
  num incorrect                    |      198146 |     7644251 |    10139821 |     2051318
  misprediction rate               |       0.12% |       5.27% |       2.94% |       0.81%
  mpki                             |        0.12 |        3.03 |        3.60 |        0.70
TLB Summary                        |             |             |             |            
  I-TLB                            |             |             |             |            
    num accesses                   |   221594606 |   286309035 |   387114354 |   358982327
    num misses                     |        1247 |       14272 |        4145 |       12409
    miss rate                      |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                           |        0.00 |        0.01 |        0.00 |        0.00
  D-TLB                            |             |             |             |            
    num accesses                   |   552759375 |   821120696 |   815276841 |   847266912
    num misses                     |      216199 |     3780127 |      231519 |     3499177
    miss rate                      |       0.04% |       0.46% |       0.03% |       0.41%
    mpki                           |        0.13 |        1.50 |        0.08 |        1.19
  L2 TLB                           |             |             |             |            
    num accesses                   |      217446 |     3794399 |      235664 |     3511586
    num misses                     |      129286 |      528259 |       83977 |      501435
    miss rate                      |      59.46% |      13.92% |      35.63% |      14.28%
    mpki                           |        0.08 |        0.21 |        0.03 |        0.17
Cache Summary                      |             |             |             |            
  Cache L1-I                       |             |             |             |            
    num cache accesses             |   221594606 |   286309035 |   387114354 |   358982327
    num cache misses               |      147312 |    14758213 |      360957 |     6846185
    miss rate                      |       0.07% |       5.15% |       0.09% |       1.91%
    mpki                           |        0.09 |        5.85 |        0.13 |        2.32
  Cache L1-D                       |             |             |             |            
    num cache accesses             |   552759375 |   821120696 |   815276841 |   847266912
    num cache misses               |     8798274 |    16192813 |     7209900 |    15406401
    miss rate                      |       1.59% |       1.97% |       0.88% |       1.82%
    mpki                           |        5.15 |        6.42 |        2.56 |        5.23
  Cache L2                         |             |             |             |            
    num cache accesses             |    12871968 |    31879043 |     9176344 |    24232233
    num cache misses               |     6721248 |     6010208 |     4006080 |     6397326
    miss rate                      |      52.22% |      18.85% |      43.66% |      26.40%
    mpki                           |        3.94 |        2.38 |        1.42 |        2.17
DRAM summary                       |             |             |             |            
  num dram accesses                |    40230077 |           0 |           0 |           0
  average dram access latency (ns) |      107.67 |         inf |         inf |         inf
  average dram queueing delay      |      107.67 |        0.00 |        0.00 |        0.00
Coherency Traffic                  |             |             |             |            
  num loads from dram              |     5245472 |     4350005 |     3488433 |     5365421
  num loads from dram cache        |           0 |           0 |           0 |           0
  num loads from remote cache      |           0 |           0 |           0 |           0
